File: codeforces/problemset/1519A.py
N = int(input())
for case in range(N):
    a, b, d = map(int, input().split())

    if abs(a - b) <= d:
        print('yes')
    else:
        # greedy, spread minimum color
        minbeans = min(a, b)

        # max allowable d
        dtotal = minbeans * d

        # need to pair beans
        mxbeans = max(a, b)
        mxbeans -= minbeans

        # beans to play with
        if mxbeans <= dtotal:
            print('yes')
        else:
            print('no')

File: codeforces/problemset/1519A.cpp
#include <iostream>
#include <algorithm>
using namespace std;
typedef long long ll;

void solve() {
  ll a, b, d;
  cin >> a >> b >> d;

  ll dtotal = min(a, b) * d;
  ll leftover = abs(a - b);
  cout << (leftover <= dtotal ? "YES" : "NO") << endl;
}

int main() {
  int N;
  cin >> N;
  for (int i = 0; i < N; i++) {
    solve();
  }
}
